### PR TITLE
Add regression test for issue #835 so it correctly tests type pretty printer.

### PR DESCRIPTION
**Pull request by taimoorzaeem**
Fixes issue #1329.

### CODE REVIEW

**Comment by RyanGlScott** [comment]
Thanks! I think the CI is failing due to an issue that was [recently fixed on `master`](https://github.com/GaloisInc/cryptol/pull/1414). Can you rebase this PR on top of `master`?

**Comment by taimoorzaeem** [comment]
Yeah sure! Done.

File: tests/issues/issue835.icry.stdout [hunk 1]
```diff
@@ -1,9 +1,4 @@
 Loading module Cryptol
-Loading module Cryptol
```
Comment (RyanGlScott) [inline tests/issues/issue835.icry.stdout:2]: The [CI is failing on this test with](https://github.com/GaloisInc/cryptol/runs/8023957727?check_suite_focus=true#step:9:136):

```
      issue835.icry: [Failed]
1a2
> Loading module Cryptol

# If output is OK:
cp /home/runner/work/cryptol/cryptol/output/./tests/issues/issue835.icry.stdout \
     /home/runner/work/cryptol/cryptol/tests/issues/issue835.icry.stdout
```

This patch deletes one of the `Loading module Cryptol` lines, but I think both are required. Indeed, if I run `issue835.icry` locally, I get:

```
Loading module Cryptol
Loading module Cryptol
Loading module Float
number`{1, Float _ _} :
  {n, m} (ValidFloat n m, Literal 1 (Float n m)) => Float n m
```

One of those `Loading module Cryptol`s comes from starting up Cryptol, and another comes from `:m Float`.

Comment (taimoorzaeem) [reply tests/issues/issue835.icry.stdout:2]: Thanks for pointing this out. I am going to close this pull request and make a new pull request including these changes.

**Pull request closed without merging**